websites/S/Sofi/presence.ts

```diff
@@ -1,113 +1,130 @@
 const presence = new Presence({
 		clientId: "1028679580027977839",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	content = {
+		viewingImage: "https://i.imgur.com/9WPWqvT.png",
+		buyingImage: "https://i.imgur.com/JKLNlvT.png",
+		viewingImageText: "Viewing...",
+		buyingImageText: "Buying...",
```

Comment:
Must also start with a capital letter so ViewingImage etc

Reply:
are you for real? Using camel case is always preffered.

Reply:
Tbh idk why they recommended me it. So maybe @EncryptedDev can explain

Reply:
There seems to be a little bit of consuion here. We prefer [Enums](https://www.typescriptlang.org/docs/handbook/enums.html) for `Assets` because Enums are designed to for set named constants. The standard for naming Enum values is the use of TitleCase (e.g. `ViewingImage` or `BuyingImage`). 
I see where you're coming from, `camelCase` as you've done in teh quoted section above is correct, but we aren't talking about Enums here

Reply:
okay I'll make the changes.